README.md

```diff
@@ -64,6 +64,28 @@ Then run:
 
 So you can `ssh myserver.com:5202` to ssh to your NAS.
 
+4. If you want to run server as service you can follow the steps given below:
+
+##### On Linux (Systemd):
+
+a. Create a service file:
+```bash
+wget https://github.com/rapiz1/rathole/blob/main/examples/systemd/rathole@.service # download the file
+sudo cp rathole@.service /lib/systemd/system/rathole.service
```

Comment:
`@` mark has special meaning for systemd and shouldn't be removed at will. You may want to checkout https://superuser.com/questions/393423/the-symbol-and-systemctl-and-vsftpd and man pages.